### PR TITLE
Feature: Localization with Crowdin

### LINKED ISSUE #40: Website Translation
Hello guys, your website is only available in English and I want to translate it into my native Georgian and use it as my default language. I think that many Georgians will be happy with such news on your official website and many will use it and easily understand everything.

### PR DESCRIPTION
**Pull request by TheOnlyTails**
## Description

Added internationalization with [svelte-i18n](https://github.com/kaisermann/svelte-i18n/). This provides an `_` import, which can be used together with Svelte's `$` subscription syntax for stores to access the localization data.
All files related to internationalization are placed in https://github.com/files-community/website/tree/localization/src/i18n/, aliased as `$i18n` for use in imports.

To add a new locale, create a file in https://github.com/files-community/website/tree/localization/src/i18n/locales/ with the `[locale].json` format, copy the contents of `en.json` into it, and start translating.

Translating is done by editing the locale file, or via [Crowdin](https://crowdin.com/project/files-website).

> **Note**
> This PR, to the request of @yaira2, doesn't localize any docs pages or blog posts.
> It only applies to navigational and static content elements, such as the navbar, the footer, and the home page.

## Motivation and Context

Closes #40

## Screenshots (if appropriate)

image

### CODE REVIEW

**Comment by yaira2** [comment]
How do we set this up with Crowdin?

**Comment by TheOnlyTails** [comment]
I believe we need to create a new project for the website on Crowdin, then connect it to this repo with their GitHub app.

File: src/i18n/locales/en.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
+{
+	"metadata": {
+		"name": "Files",
```
Comment (yaira2) [inline src/i18n/locales/en.json:3]: The app name should probably be kept in English

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**